Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Keep version in-sync with .travis.yml
```

Comment:
Nit: Should we add 'and app/.travis.yml' ?

Reply:
Not sure if `app/.travis.yml` is used for anything. I wanted to check it and remove, but never got around it.

Reply:
:)

Reply:
FYI - https://github.com/dart-lang/mono_repo

But that tool has been updated to use a new kind of file.

I'll fix...